src/lightning/store/utils.py

```diff
@@ -0,0 +1,88 @@
+# Copyright The Lightning team.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import json
+import os
+from enum import Enum
+
+
+class stage(Enum):
+    UPLOAD = 0
+    LOAD = 1
+    DOWNLOAD = 2
+
+
+_LIGHTNING_DIR = os.path.join(os.path.expanduser("~"), ".lightning")
+_LIGHTNING_STORAGE_FILE = os.path.join(_LIGHTNING_DIR, ".lightning_model_storage")
+_LIGHTNING_STORAGE_DIR = os.path.join(_LIGHTNING_DIR, "lightning_model_store")
+_LIGHTNING_CLOUD_URL = os.getenv("LIGHTNING_CLOUD_URL", default="https://lightning.ai")
```

Comment:
These can (/ already do in cases) exist in constants :smiley:

Reply:
done :+1: